### PR TITLE
New

### PR DESCRIPTION
**Pull request by Yochyo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dsadsadassadsadsa
 # Minecraft Lasertag mod
```
Comment (Yochyo) [inline README.md:2]: fdsfdsfdsfds

**Comment by efibs** [comment]
Terrible code lol

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+dsadsadassadsadsa
```
Comment (efibs) [inline README.md:1]: Why!?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 # Minecraft Lasertag mod
 
 ![GitHub license](https://img.shields.io/github/license/efibs/mc-lasertag-mod?style=flat-square)
```
Comment (Yochyo) [inline README.md:4]: dsadsa

File: README.md (hunk 2 above)
Comment (Yochyo) [inline README.md:1]: dfsdsadsa

Comment (efibs) [inline README.md:1]: Why!?

**Review by efibs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**